client/jetpack-cloud/sections/partner-portal/primary/payment-method-list/index.tsx

```diff
@@ -12,13 +13,112 @@ import Main from 'calypso/components/main';
 import CardHeading from 'calypso/components/card-heading';
 import DocumentHead from 'calypso/components/data/document-head';
 import SidebarNavigation from 'calypso/jetpack-cloud/sections/partner-portal/sidebar-navigation';
-import { Button } from '@automattic/components';
+import { getAllStoredCards } from 'calypso/state/stored-cards/selectors';
+import QueryStoredCards from 'calypso/components/data/query-stored-cards';
+import {
+	VisaLogo,
+	MastercardLogo,
+	AmexLogo,
+} from 'calypso/my-sites/checkout/composite-checkout/components/payment-logos';
+import EllipsisMenu from 'calypso/components/ellipsis-menu';
+import PopoverMenuItem from 'calypso/components/popover/menu-item';
+import Gridicon from 'calypso/components/gridicon';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+function StoredCard( props ): ReactElement {
```

Comment:
We've been storing this kind of component in `components`. So I think we can move them there, then import and use them here.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/53416/commits/39471a9d7e93ff6e2566769c1ee4d00b0ef4bb46 , thanks!